### PR TITLE
voting phase

### LINKED ISSUE #1747: Retro Frontend: Voting
## Issue - Enhancement

This story adds the ability to vote on Reflection Groups for retro meetings, on the frontend.

### Acceptance Criteria (optional)
- Team Members see all Reflections and Reflection Groups
- Team Members see their available votes
- Team Members can add their vote to a Reflection or Reflection Group
- Team Members can remove their vote from a Reflection or Reflection Group
- Team Members can only see their own votes
- Team Members cannot vote once the Voting Phase is no longer the active Phase
- Once the Vote Phase starts, all ungrouped Reflections are turned into one-item Groups to simplify voting logic ([see Figma design for details](https://www.figma.com/file/ClYTi13FvvTZ2y03Om8wNrXq/Retro-Views))

- **Estimated effort:** 13 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by mattkrick**
note: ignore the branch name, the voting phase needed a little extra love & i didn't want to have a PR reviewed & merged to master that wasn't prod quality.

also, if you try to vote & encounter an error, just hit refresh & it'll work. this is fixed in the new discuss phase. i'm sure there are other little bugs i'll fix up along the way...

fixes #1747

TEST
- [ ] Go to the voting phase, can expand groups, but cannot drag cards
- [ ] can click the gray checkmark up to 3 times
- [ ] can remove votes by clicking pink checkmarks
- [ ] see updated vote info in the sidebar
- [ ] Browser B (different user) sees updated team votes remaining, but receives no user-specific info
- [ ] groups with only 1 reflection has an auto-generated title

### CODE REVIEW

File: src/server/graphql/mutations/helpers/addEntitiesToReflections.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const addEntitiesToReflections = async (meetingId) => {
 
   const nextReflections = reflections.map((reflection, idx) => ({
     id: reflection.id,
-    entities: sanitizedReflectionResponses[idx].entities
+    entities: sanitizedReflectionResponses[idx]
```
Comment (mattkrick) [inline src/server/graphql/mutations/helpers/addEntitiesToReflections.js]: supports failed google lookups

File: src/server/graphql/mutations/helpers/createNewMeetingPhases.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ const createNewMeetingPhases = async (teamId, meetingId, meetingCount, meetingTy
       return {
         id: shortid.generate(),
         phaseType,
-        stages: []
+        stages: [makeRetroStage(phaseType, meetingId)]
```
Comment (mattkrick) [inline src/server/graphql/mutations/helpers/createNewMeetingPhases.js]: creates a placeholder topic that will be overwritten once the votes are tallied

File: src/universal/components/NewMeetingSidebarPhaseListItem.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
+import type {Node} from 'react';
 // @flow
 import React from 'react';
 import styled, {css} from 'react-emotion';
 import appTheme from 'universal/styles/theme/appTheme';
 import ui from 'universal/styles/ui';
+import {phaseLabelLookup} from 'universal/utils/meetings/lookups';
 
 const NavListItem = styled('li')({
-  fontSize: 0,
```
Comment (mattkrick) [inline src/universal/components/NewMeetingSidebarPhaseListItem.js]: hey @ackernaut what's the purpose of fontSize: 0? i had to remove it here, but i saw it in other places. just curious.

**Comment by mattkrick** [comment]
@jordanh ready for review!

File: src/universal/components/NewMeetingSidebarPhaseListItem.js (hunk 3 above)
Comment (ackernaut) [reply src/universal/components/NewMeetingSidebarPhaseListItem.js]: It was an artifact from using `inline-block` to lay inner elements out LTR. We should be using flex if we aren’t already to handle this.

**Comment by mattkrick** [comment]
closing in favor of #1986, this is good for code review, but #1986 has the full experience

**Pull request closed without merging**